code/modules/client/preference_setup/loadout/loadout_suit_vr.dm

```diff
@@ -32,4 +32,14 @@
 
 /datum/gear/suit/gntop
 	display_name = "GN crop jacket"
-	path = /obj/item/clothing/suit/storage/fluff/gntop
\ No newline at end of file
+	path = /obj/item/clothing/suit/storage/fluff/gntop
+
+/datum/gear/suit/old_poncho //This is made from an old sprite which has been here for quite some time. Called old poncho because duplicates
+	display_name = "Colorful poncho"
+	description = "A mexican looking poncho. It look like it fits wolf taurs as well."
+	path = /obj/item/clothing/suit/poncho
+
+/datum/gear/suit/old_poncho //This is made from an old sprite which has been here for quite some time. Called old poncho because duplicates
+	display_name = "Colorful poncho"
+	description = "A mexican looking poncho. It look like it fits wolf or drake taurs as well."
```

Comment:
defined twice for some reason, with same path of datum and clothing?

Reply:
I am not sure what you mean, but there is other ponchos in the game. Particularly for /datum/gear/suit/poncho, so i used old_poncho instead

Reply:
There are two listings of /datum/gear/suit/old_poncho, both of them are named exactly the same and have the same contents?

Reply:
Oh, that's weird.